src/Account.ts

```diff
@@ -395,7 +398,7 @@ export class Account {
     if (params.transactionCallback) {
       announcer.vaultTransaction.monitor((event: any) => {
         const address: string = this.reachStdLib.formatAddress(event.what[0]);
-        const rawVaultState = event.what[1];
+        const rawVaultState = event.what[2];
```

Comment:
This will require a new version update to take effect @Jesulonimi21 - is it a bug?